packages/react/src/components/Tile/Tile.js

```diff
@@ -21,190 +21,176 @@ import { composeEventHandlers } from '../../tools/events';
 
 const { prefix } = settings;
 
-export class Tile extends Component {
-  static propTypes = {
-    /**
-     * The child nodes.
-     */
-    children: PropTypes.node,
+export const Tile = ({ children, className, light = false, ...other }) => {
```

Comment:
Small style nits (almost all for codebase consistency, not correctness):

- Our style guide uses function declarations for components: https://github.com/carbon-design-system/carbon/blob/main/docs/style.md#writing-a-component
- `other` -> `rest`
- `classNames` -> `cx`